pkg/cdi/container-edits.go

```diff
@@ -113,6 +113,16 @@ func (e *ContainerEdits) Apply(spec *oci.Spec) error {
 		}
 	}
 
+	if e.NetDevices != nil {
+		// specgen is currently missing functionality to set Linux NetDevices,
+		// so we use a locally rolled function for now.
+		for _, dev := range e.NetDevices {
+			specgenAddLinuxNetDevice(&specgen, dev.HostIf, &oci.LinuxNetDevice{
```

Comment:
Is this something we want to contribute to `specgen`?

Reply:
Yes, I am planning to do it once the necessary runtime-spec bits get tagged.